src/Roslyn.Diagnostics.Analyzers/UnitTests/ApplyTraitToClassTests.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Test.Utilities;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeRefactoringVerifier<
+    Roslyn.Diagnostics.CSharp.Analyzers.CSharpApplyTraitToClass>;
+using VerifyVB = Test.Utilities.VisualBasicCodeRefactoringVerifier<
+    Roslyn.Diagnostics.VisualBasic.Analyzers.VisualBasicApplyTraitToClass>;
+
+namespace Roslyn.Diagnostics.Analyzers.UnitTests
+{
+    public class ApplyTraitToClassTests
+    {
+        [Theory]
+        [InlineData("A", "")]
+        [InlineData("", "A")]
+        [InlineData("A", "A")]
+        public async Task MoveTraitToType_MovesSecond_CSharp(string name, string value)
+        {
+            var source = $@"
+using Xunit;
+
+class C
+{{
+    [$$Trait(""{name}"", ""{value}"")]
+    public void Method() {{ }}
+
+    [Fact, Trait(""{name}"", ""{value}"")]
+    public void Method2() {{ }}
+}}
+";
+            var fixedSource = $@"
+using Xunit;
+
+    [Trait(""{name}"", ""{value}"")]
+class C
```

Comment:
Can the refactoring put the attribute with correct indentation?

Reply:
I'm not sure why it does this. 😆

Reply:
If SyntaxGenerator is supposed to add the attribute in proper formatting. Then it's most likely a SyntaxGenerator bug. Otherwise, I think we can annotate the node to be formatted.

Reply:
I'll add a formatting annotation and see if it fixes it.

Reply:
⏱ Formatter annotation fixed it